packages/components/src/Stack/Stack.tsx

```diff
@@ -10,21 +10,32 @@ export type StackProps = {
   title?: string; // For Testing
 };
 
```

Comment:
Ups ... this shouldn't be here. But fine I guess?

Reply:
Ok for me